docs/source/release_notes.rst

```diff
@@ -17,7 +33,179 @@ Future Release
         * Update boto3 and urllib3 version requirements (:pr:`1394`)
 
     Thanks to the following people for contributing to this release:
-    :user:`gsheni`, :user:`rwedge`, :user:`thehomebrewnerd`
+    :user:`gsheni`, :user:`jeff-hernandez`, :user:`rwedge`, :user:`tamargrey`, :user:`thehomebrewnerd`
+
+Breaking Changes
+++++++++++++++++
+
+* ``Entity.add_interesting_values`` has been removed. To add interesting values for a single
+  entity, call ``EntitySet.add_interesting_values`` and pass the name of the dataframe for
+  which to add interesting values in the ``dataframe_name`` parameter.
+* ``Entity.set_secondary_time_index`` has been removed and replaced by ``EntitySet.set_secondary_time_index``
+  with an added ``dataframe_name`` parameter to specify the dataframe on which to set the secondary time index.
+* ``Relationship`` initialization has been updated to accept four name values for the parent dataframe,
+  parent column, child dataframe and child column instead of accepting two ``Variable`` objects.
+* ``EntitySet.add_relationship`` has been updated to accept dataframe and column name values or a
+  ``Relationship`` object. Adding a relationship from a ``Relationship`` object now requires passing
+  the relationship as a keyword argument.
+* ``Entity.update_data`` has been removed. To update the dataframe, call ``EntitySet.update_dataframe`` and use the ``dataframe_id`` paramter.
+* The data in an ``EntitySet`` is no longer stored in ``Entity`` objects. Instead, dataframes
+  with Woodwork typing information are used. Accordingly, most language referring to “entities”
+  will now refer to “dataframes”, references to “variables” will now refer to “columns”, and
+  “variable types” will use the Woodwork type system’s “logical types” and “semantic tags”
+* The dictionary of tuples passed to ``EntitySet.__init__`` has replaced the ``variable_types`` element
+  with separate ``logical_types`` and ``semantic_tags`` dictionaries.
+* ``EntitySet.entity_from_dataframe`` no longer exists. To add new tables to an entityset, use``EntitySet.add_dataframe``.
+* ``EntitySet.normalize_entity`` has been renamed to ``EntitySet.normalize_dataframe``
+* Instead of raising an error at ``EntitySet.add_relationship`` when the dtypes of parent and child columns
+  do not match, featuretools will now check whether the Woodwork logical type of the parent and child columns
+  match. If they do not match, there will now be a warning raised, and featuretools will attempt to update
+  the logical type of the child column to match the parent’s.
+* If no index is specified at ``EntitySet.add_dataframe``, the first column will only be used as index if
+  Woodwork has not been initialized on the DataFrame. When adding a dataframe that already has Woodwork
+  initialized, if there is no index set, an error will be raised.
+* Due to restrictions on Woodwork initialization, Koalas DataFrames can no longer use the make_index parameter.
```

Comment:
@thehomebrewnerd this is no longer the case with the `make_index` changes merged in recently, right?

Reply:
Correct, we do now support make_index with Koalas.